Content.Server/DeltaV/Paper/SignatureSystem.cs

```diff
@@ -0,0 +1,106 @@
+using Content.Server.Access.Systems;
+using Content.Server.Paper;
+using Content.Server.Popups;
+using Content.Shared.Paper;
+using Content.Shared.Popups;
+using Content.Shared.Tag;
+using Content.Shared.Verbs;
+using Robust.Server.Audio;
+using Robust.Shared.Player;
+
+namespace Content.Server.DeltaV.Paper;
+
+public sealed class SignatureSystem : EntitySystem
+{
+    [Dependency] private readonly AudioSystem _audio = default!;
+    [Dependency] private readonly IdCardSystem _idCard = default!;
+    [Dependency] private readonly PaperSystem _paper = default!;
+    [Dependency] private readonly PopupSystem _popup = default!;
+    [Dependency] private readonly TagSystem _tagSystem = default!;
+
+    // The sprite used to visualize "signatures" on paper entities.
+    private const string SignatureStampState = "paper_stamp-signature";
```

Comment:
Is this validatable?

Reply:
Pretty sure stamp states cannot be validated, although I might be wrong.

Reply:
Yeah, just an arbitrary RSI state, no validation